tests/php/ORM/DataListEagerLoadingTest.php

```diff
@@ -897,7 +898,7 @@ public function testEagerLoadingFilteredList(string $iden, array $eagerLoad): vo
     {
         $this->createEagerLoadData(5);
         $filter = ['Title:GreaterThan' => 'obj 0'];
-        $dataList = EagerLoadObject::get()->filter($filter)->eagerLoad(...$eagerLoad);
+        $dataList = EagerLoadObject::get()->filter($filter)->eagerLoad($eagerLoad);
```

Comment:
I've swapped a few existing tests to pass the array directly, so we have coverage for non-associative arrays. I've left others alone to retain coverage for passing strings.